packages/kernel-browser-runtime/src/kernel-worker/kernel-worker.ts

```diff
@@ -39,26 +31,23 @@ async function main(): Promise<void> {
     (listener) => globalThis.removeEventListener('message', listener),
   );
 
-  const kernelStream = await MessagePortDuplexStream.make<
-    JsonRpcCall,
-    JsonRpcResponse
-  >(port, isJsonRpcCall);
-
   // Initialize kernel dependencies
-  const vatWorkerClient = VatWorkerClient.make(globalThis as PostMessageTarget);
-  const kernelDatabase = await makeSQLKernelDatabase({
-    dbFilename: DB_FILENAME,
-  });
+  const [kernelStream, vatWorkerClient, kernelDatabase] = await Promise.all([
+    MessagePortDuplexStream.make<JsonRpcCall, JsonRpcResponse>(
+      port,
+      isJsonRpcCall,
+    ),
+    VatWorkerClient.make(globalThis as PostMessageTarget),
+    makeSQLKernelDatabase({ dbFilename: DB_FILENAME }),
+  ]);
   const firstTime = !kernelDatabase.kernelKVStore.get('initialized');
 
   const kernel = await Kernel.make(
     kernelStream,
     vatWorkerClient,
     kernelDatabase,
-    {
-      resetStorage: ALWAYS_RESET_STORAGE,
-    },
```

Comment:
Is there still some way to reset storage without reinstalling the extension? If not, there should be, but it can be done in a follow-up.

(Reinstalling a locally unpacked extension doesn't always successfully clear its storage IME.)